### PR TITLE
Fixed some code smells in apollo-portal module

### PR DESCRIPTION
**Pull request by WillardHu**
## What's the purpose of this PR

- Inline variable
- Chenge deprecated method
- Add `final` or `static final` keyword to class field

## Which issue(s) this PR fixes:
Fixes #

## Brief changelog

fixed some code smells

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [ ] Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by WillardHu** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/util/OpenApiBeanUtils.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
 public class OpenApiBeanUtils {
 
   private static final Gson GSON = new Gson();
-  private static Type type = new TypeToken<Map<String, String>>() {}.getType();
+  private static Type TYPE = new TypeToken<Map<String, String>>() {}.getType();
```
Comment (kezhenxu94) [inline apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/util/OpenApiBeanUtils.java:54]:
```suggestion
  private static final Type TYPE = new TypeToken<Map<String, String>>() {}.getType();
```

File: apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/NamespaceBranchController.java [hunk 2]
```diff
@@ -49,6 +45,9 @@
 
 import javax.servlet.http.HttpServletRequest;
 
+/**
+ * Created by qianjie on 8/10/17.
+ */
```
Comment (kezhenxu94) [inline apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/NamespaceBranchController.java:50]: Remove this

File: apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/AppController.java [hunk 3]
```diff
@@ -84,7 +84,7 @@ public AppController(final PortalSettings portalSettings,
   @GetMapping("/apps")
   public List<OpenAppDTO> findApps(@RequestParam(value = "appIds", required = false) String appIds) {
     final List<App> apps = new ArrayList<>();
-    if (StringUtils.isEmpty(appIds)) {
+    if (StringUtils.hasLength(appIds)) {
```
Comment (kezhenxu94) [inline apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/AppController.java:87]: The logical result is reversed, which is wrong

```suggestion
    if (!StringUtils.hasLength(appIds)) {
```

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]

**Review by kezhenxu94 — APPROVED** [review]

**Comment by WillardHu** [comment]
> WillardHu force-pushed the WillardHu:fix/code-smells branch from 80716c6 to b66b78b 21 seconds ago

rebase master and retest with jdk8 unit-test

**Review by kezhenxu94 — DISMISSED** [review]
LGTM

**Comment by WillardHu** [comment]
> WillardHu force-pushed the WillardHu:fix/code-smells branch from b66b78b to 0f95412 2 minutes ago

Fixed the ConsumerAuditUtil becuse ConsumerAuditUtilTest used reflection to set `BATCH_TIMEOUT` and `BATCH_TIMEUNIT`, so without `final`. Passed unit-test in local.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**